platform/android/java_godot_io_wrapper.cpp

```diff
@@ -118,6 +119,19 @@ int GodotIOJavaWrapper::get_screen_dpi() {
 	}
 }
 
+void GodotIOJavaWrapper::get_window_safe_area(int (&p_rect_xywh)[4]) {
+	if (_get_window_safe_area) {
+		JNIEnv *env = ThreadAndroid::get_env();
+		jintArray returnArray = (jintArray)env->CallObjectMethod(godot_io_instance, _get_window_safe_area);
+		jint *arrayBody = env->GetIntArrayElements(returnArray, JNI_FALSE);
+		ERR_FAIL_COND(env->GetArrayLength(returnArray) != 4);
```

Comment:
Maybe it can be actually before `GetIntArrayElements`, so that if it fails and returns it wouldn't leak memory from the array. Shouldn't happen anyway but it kind of hurts me to keep it with a potential leak :)

Reply:
Also, for small changes like that, you can squash commits together right away, otherwise someone is going to ask you to do it later anyway.